.circleci/config.yml

```diff
@@ -56,31 +18,69 @@ jobs:
           paths:
             - project
             - scripts
-  01-login:
+  sample-01:
     machine: true
     environment:
-      - AUTH0_CFG: 01-Login/auth_config.json
-      - SAMPLE_PATH: 01-Login
-    <<: *defaults
+      - AUTH0_CFG: Sample-01/auth_config.json
```

Comment:
I know this is merging 2 samples into 1, but let's check with someone else if we should name them "sample-01" instead of following the previous style.

Reply:
Sure but I believe it was discussed when we did the React one, which does the same thing as this.

It didn't make sense to call it '01-Login' when the sample actually does much more than just login.